app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/network/MarvelApi.kt

```diff
@@ -0,0 +1,17 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.network
```

Comment:
packages are not for navigation but for grouping things in units, you have great IDE to navigate between files.